database/mysql_anonymiser.go

```diff
@@ -61,6 +54,37 @@ func (a *MySQLAnonymiser) DumpTable(table string, rowChan chan<- []*Cell, endCha
 	return nil
 }
 
+// - If the replacement is a literal, then we simply place it in the Cell. No faking here.
+// - If the replacement is not a literal, and it is an actual replacement then call the faker function
+// that it maps on to.
+// - We do not anonymise fields in columns that are not listed in config.
+func (a *MySQLAnonymiser) anonymiseCellv2(column, replacement string) *Cell {
```

Comment:
just rename this and remove the other one

Reply:
Done.